tests/rpm_tests/test_create_rpm.py

```diff
@@ -382,6 +383,10 @@ def specific_file_exclude(name, path):
     def src_ignore_exclude(path):
         return path not in IGNORED_SOURCE_FILES
 
+    # TODO: This is just a temporary workaround because JS has too many files to ignore
+    @staticmethod
+    def cockpit_package_exclude(rpm):
+        return "ui/cockpit" not in rpm
```

Comment:
We should make sure that there are no files with the new UI in the RPMs, shouldn't we?

Reply:
Good point. That is test which should be there but could be done later after this is merged.

Reply:
I have created an issue for that.